caiman/base/traces.py

```diff
@@ -1,25 +1,19 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-"""
-author: agiovann
-"""
-#%%
+
 import cv2
 import logging
 import numpy as np
 import pylab as pl
 pl.ion()
-from . import timeseries as ts
+
+import caiman.base.timeseries
```

Comment:
in general with `caiman.base` I would prefer to import the final object. E.g., just import `timeseries` because `caiman.base.timeseries.timeseries` doesn't help me read the code.